### PR TITLE
chore: add health e2e test

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

Add health e2e test that we don't have before.
image

### CODE REVIEW

**Comment by starsz** [comment]
cc @Jaycean Can you review my code when you are free?

**Review by nic-chen — APPROVED** [review]

**Comment by Jaycean** [comment]
> cc @Jaycean Can you review my code when you are free?

no problem.

**Review by Jaycean — APPROVED** [review]
LGTM.

File: api/test/e2enew/healthz/healthz_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+package healthz
+
+import (
+	"testing"
+
+	"github.com/onsi/ginkgo"
+)
+
+func TestRoute(t *testing.T) {
```
Comment (tokers) [inline api/test/e2enew/healthz/healthz_suite_test.go:25]: It has nothing to do with `Route`, the title is confusing, man.

File: api/test/e2enew/healthz/healthz_test.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * limitations under the License.
+ */
+package healthz
+
+import (
+	"net/http"
+
+	"github.com/onsi/ginkgo"
+
+	"e2enew/base"
```
Comment (tokers) [inline api/test/e2enew/healthz/healthz_test.go:24]: We'd better add the module prefix, or we may think the "e2enew/base" is a standard package.

File: api/test/e2enew/healthz/healthz_suite_test.go (hunk 1 above)
Comment (starsz) [reply api/test/e2enew/healthz/healthz_suite_test.go:25]: Sorry, my mistake.

File: api/test/e2enew/healthz/healthz_test.go (hunk 2 above)
Comment (starsz) [reply api/test/e2enew/healthz/healthz_test.go:24]: OK. I will fix it after all tests PR merged.

Comment (starsz) [reply api/test/e2enew/healthz/healthz_test.go:24]: I will create another PR to do this work for all tests.

**Pull request merged**